cfgov/cfgov/urls.py

```diff
@@ -289,6 +300,16 @@
     url(r'^token-provider/', token_provider),
 ]
 
+with flagged_urls('CCDB5_RELEASE') as _url:
+    apiBase = '^data-research/consumer-complaints/api/v1'
+    ccdb5_patterns = [
+        _url(apiBase + '/_suggest', CCDB5.suggest),
+        _url(apiBase + '/(?P<id>[0-9]+)$', CCDB5.document),
+        _url(apiBase, CCDB5.search)
+    ]
+urlpatterns += ccdb5_patterns
```

Comment:
Because of the use of a fallback view for `^data-research/consumer-complaints/`… not really right now. It would be nice, and I'll happily take suggestions on how to make it so. 

It's possible with https://github.com/cfpb/wagtail-flags/pull/13 we could use a pair of includes for `^data-research/consumer-complaints/`, and fallback on the current in `with flagged_urls('CCDB5_RELEASE', fallback=include(old_ccdb_urls)) as url`. But this is about the only way to do these right now.